src/Form.js

```diff
@@ -0,0 +1,124 @@
+import {useState} from 'react';
+import Workout from './Workout';
+
+const Form = ({exercises, setExercises}) => {
+	const [userFilters, setUserFilters] = useState({
+		howMany: 0,
+		abs: false,
+		arms: false,
+		back: false,
+		chest: false,
+		legs: false,
+		shoulders: false,
+	});
+	const [filteredExercises, setFilteredExercises] = useState([]);
+
+	function handleChange(event) {
+		const {name, value, type, checked} = event.target;
+		setUserFilters((prev) => {
+			return {
+				...prev,
+				[name]: type === 'checkbox' ? checked : value,
+			};
+		});
+	}
+
+	const getWorkout = (event) => {
+		event.preventDefault();
+
+		let newArray = [];

```

Comment:
This is great! Good job! 

Maybe post-bootcamp, this could be a good opportunity to refactor this in the event that you want to be more detailed with the types of workouts a user could want (ie. biceps, triceps, forearms). But for tomorrow, don't bother!